### PR TITLE
08MQ reproduction

### PR DESCRIPTION
**Pull request by bclenet**
This Pull Request is related to issue #120 

Changes proposed in this Pull Request:

- Pipeline reproduction

Checklist:

- [x] Descriptive title
- [x] Targets the `main` branch
- [ ] Changes are functional
- [ ] My code is explicit and comments were added to it
- [ ] Code conforms with PEP8
- [ ] Tests were added for the changes and they complete successfully
- [ ] Existing tests were updated (if needed) and they complete successfully
- [ ] Documentation was updated

### CODE REVIEW

**Comment by cmaumet** [comment]
@bclenet - I have turned this PR into "ready for review" so that I can use the github interface to do a review...

**Review by cmaumet — COMMENTED** [review]
@bclenet: I've reviewd the pipeline code as suggested and did not spot any issues. One thing that would help me doing a second check would be to get the corresponding fsf file and open it into FSL. Is this feasible with nipype? (I think the equivalent is possible for SPM but I am unsure this is available for FSL). 

I have also answered in this review your 3 questions, when can continue the discussion directly in the comments.

Let me know if there is any other check I can help with!

File: narps_open/pipelines/team_08MQ.py [hunk 1]
```diff
@@ -0,0 +1,977 @@
...
+        with open(event_file, 'rt') as file:
+            next(file)  # skip the header
+
+            for line in file:
+                info = line.strip().split()
+                onsets['event'].append(float(info[0]))
+                durations['event'].append(float(info[1]))
+                amplitudes['event'].append(1.0)
+                onsets['gain'].append(float(info[0]))
+                durations['gain'].append(float(info[4])) # TODO : change to info[1] (= 4) ?
```
[the remark this answers is not in the document]
Comment (bclenet) [reply narps_open/pipelines/team_08MQ.py:406]: Sure, I far as I understand, it is not mentioned in the description. Here is a link to it : https://notes.inria.fr/kexoh3PvRYezP5fK5cg2sw#

File: narps_open/pipelines/team_08MQ.py [hunk 2]
```diff
@@ -0,0 +1,977 @@
...
+        merge_copes.inputs.dimension = 't'
+
+        # Merge Node - Merge varcopes files for each subject
+        merge_varcopes = Node(MergeImages(), name = 'merge_varcopes')
+        merge_varcopes.inputs.dimension = 't'
+
+        # FLAMEO Node - Estimate model
+        estimate_model = Node(FLAMEO(), name = 'estimate_model')
+        estimate_model.inputs.run_mode = 'fe' # Fixed effect
+        estimate_model.inputs.mask_file = Info.standard_image('MNI152_T1_2mm_brain_mask.nii.gz')
```
[the remark this answers is not in the document]
Comment (bclenet) [reply narps_open/pipelines/team_08MQ.py:608]: Ok thanks, let's do this !

File: narps_open/pipelines/team_08MQ.py [hunk 3]
```diff
@@ -0,0 +1,977 @@
...
+
+        # FLAMEO Node - Estimate model
+        estimate_model = Node(FLAMEO(), name = 'estimate_model')
+        estimate_model.inputs.run_mode = 'ols' # Ordinary least squares
+        estimate_model.inputs.mask_file = Info.standard_image('MNI152_T1_2mm_brain_mask.nii.gz')
+
+        # Randomise Node -
+        randomise = Node(Randomise(), name = 'randomise')
+        randomise.inputs.num_perm = 10000
+        randomise.inputs.tfce = True
```
[the remark this answers is not in the document]
Comment (bclenet) [reply narps_open/pipelines/team_08MQ.py:784]: Description mentions
> statistic_type : TFCE. 

Here is the link to the description https://notes.inria.fr/kexoh3PvRYezP5fK5cg2sw#

**Comment by bclenet** [comment]
I can get fsf files for the run level analysis (as soon as it runs...) : it is the output of the `model_design` ([Level1Design](https://nipype.readthedocs.io/en/latest/api/generated/nipype.interfaces.fsl.model.html#level1design)) node. Would that help ?

File: narps_open/pipelines/team_08MQ.py [hunk 4]
```diff
@@ -0,0 +1,977 @@
...
+        randomise.inputs.vox_p_values = True
+        randomise.inputs.c_thresh = 0.05
+        randomise.inputs.tfce_E = 0.01
+        randomise.inputs.mask = Info.standard_image('MNI152_T1_2mm_brain_mask.nii.gz')
```
[the remark this answers is not in the document]
Comment (bclenet) [reply narps_open/pipelines/team_08MQ.py:788]: Erratum : the FLAMEO node (couple of lines earlier) in the group level analysis uses a mandatory mask input : I guess we will have to compute the intersection of all previously computed subject-specific masks then.

File: narps_open/pipelines/team_08MQ.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmaumet) [reply narps_open/pipelines/team_08MQ.py:406]: Discussed Dec 1, we will go with 4s for the modulation regressors (same as main event)

File: narps_open/pipelines/team_08MQ.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmaumet) [reply narps_open/pipelines/team_08MQ.py:608]: Discussed on Dec 1. It looks like feat is creating the mask in the background we can check in the paper that reproduce feat in nipype how they dealt with this.

**Pull request merged**